test/test_codefix.vader

```diff
@@ -444,16 +444,6 @@ Execute("workspace/applyEdit" from LSP should be handled):
   \ [{'description': 'applyEdit', 'changes': [{'fileName': '/foo/bar/file.ts', 'textChanges': [{'end': {'offset': 28, 'line': 8}, 'newText': ', Config', 'start': {'offset': 28, 'line': 8}}, {'end': {'offset': 13, 'line': 97}, 'newText': 'await newFunction(redis, imageKey, cover, config);', 'start': {'offset': 3, 'line': 95}}, {'end': {'offset': 3, 'line': 100}, 'newText': '^@async function newFunction(redis: IRedis, imageKey: string, cover: Buffer, config: Config) {^@    try {^@        await redis.set(imageKey, cover, ''ex'', parseInt(config.coverKeyTTL, 10));^@    }^@    catch { }^@}^@', 'start': {'offset': 3, 'line': 100}}]}]}],
   \ g:code_actions
 
-Execute(Code Actions from LSP should be handled with user input if there are more than one action):
-  call ale#codefix#SetMap({2: {}})
-  call ale#codefix#HandleLSPResponse(1,
-  \ {'id': 2, 'jsonrpc': '2.0', 'result': [{'title': 'fake for testing'}, {'arguments': [{'documentChanges': [{'edits': [{'range': {'end': {'character': 31, 'line': 2}, 'start': {'character': 31, 'line': 2}}, 'newText': ', createVideo'}], 'textDocument': {'uri': 'file:///foo/bar/file.ts', 'version': 1}}]}], 'title': 'Add ''createVideo'' to existing import declaration from "./video"', 'command': '_typescript.applyWorkspaceEdit'}]})
-
-  AssertEqual g:handle_code_action_called, 1
-  AssertEqual
-  \ [{'description': 'codeaction', 'changes': [{'fileName': '/foo/bar/file.ts', 'textChanges': [{'end': {'offset': 32, 'line': 3}, 'newText': ', createVideo', 'start': {'offset': 32, 'line': 3}}]}]}],
-  \ g:code_actions
```

Comment:
I don't think this test made any sense:

1. there's just one action even though the test description says otherwise
2. it's a tsserver response so why is this being handled by `HandleLSPResponse`

Anyway I used most of it for the new test, just changed the expectation to what I think should happen, and what works with more standard language servers.

Reply:
OK